File: 0724-find-pivot-index/0724-find-pivot-index.cpp
class Solution {
public:
    int pivotIndex(vector<int>& nums) {
       int ts=0;
       int ls=0;
       for(int i=0;i<nums.size();i++){
        ts+=nums[i];
       }
       for(int i=0;i<nums.size();i++){
        int rs=ts-ls-nums[i];
        if(ls==rs)
            return i;
        ls+=nums[i];
       }
       return -1;
    }
};

File: 0724-find-pivot-index/0724-find-pivot-index.java
class Solution {
    public int pivotIndex(int[] nums) {
        
        for(int i=0;i<nums.length;i++)
        {
        int ls=0;
        int rs=0;
        for(int x=0;x<i;x++)
        {
            ls+=nums[x];
        }
        for(int j=i+1;j<nums.length;j++)
        {
            rs+=nums[j];
            

        }
        if(ls==rs)
        {
            return i;
        }

        }
        return -1;
        
    }
}